packages/web/src/components/molecules/EntityGrid/EntityGridItemDropdown.tsx

```diff
@@ -0,0 +1,83 @@
+import {useCallback, useMemo} from 'react';
+
+import {ItemType} from 'antd/lib/menu/hooks/useItems';
+
+import {useDashboardNavigate} from '@hooks/useDashboardNavigate';
+import useRunEntity from '@hooks/useRunEntity';
+
+import {DotsDropdown} from '@molecules';
+
+import {Item} from './EntityGridItemPure';
+
+type EntityGridItemDropdownProps = {
+  entityLabel: string;
+  item: Item;
+  outOfSync?: boolean;
+  onAbort: (item: Item) => void;
+};
+
+const EntityGridItemDropdown: React.FC<EntityGridItemDropdownProps> = props => {
+  const {entityLabel, item, outOfSync, onAbort} = props;
+
+  const editNavigate = useDashboardNavigate(
+    () =>
+      `/${entityLabel === 'test' ? 'tests' : 'test-suites'}/${item.name}/settings/${
+        entityLabel === 'test' ? 'test' : 'tests'
+      }`
```

Comment:
It looks like it's a part of the Cloud PR too, isn't it? I'm guessing there is no need to duplicate this file